### PR TITLE
feat: Adapt to https://github.com/interledger/rfcs/pull/177

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,6 +219,30 @@ const receiver = new FiveBellsLedgerPlugin({
...
+$ npm run test
+```
+
+To run the integration tests, which test the integration between this node modules and related ones like
+[ilp-plugin-bells](https://github.com/interledgerjs/ilp-plugin-bells), run:
+
+```sh
+$ npm run integration-loader
+$ node ./node-modules/.bin/integration setup
+$ node ./node_modules/.bin/integration test
```
Comment (emschwartz) [inline README.md]: All of this is actually done by running `npm run integration`

**Comment by emschwartz** [comment]
This should be marked as a breaking change because it will not work with older plugins

File: README.md [hunk 2]
```diff
@@ -231,17 +231,13 @@ To run the integration tests, which test the integration between this node modul
...
 $ node ./node_modules/.bin/integration test
 ```
 
-To repeat one of the integration tests, run for instance:
-
-```sh
-$ cd integration-test
-$ mocha ../node_modules/five-bells-integration-test/src/tests/index.js
-```
+After a fix, you can repeat just the last step, or just the two last steps. It will complete faster than
```
Comment (emschwartz) [inline README.md]: I think you need to run `rm -rf integration-test/data` if you want to re-run `integration test`

File: README.md (hunk 1 above)
Comment (michielbdejong) [inline README.md]: yeah, removed this again. we can now use the Docker image. :)

**Pull request merged**